src/components/PostList/PostList.jsx

```diff
@@ -1 +1,10 @@
-export const PostList = () => <>Put the list here</>;
+import './PostList.scss';
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div key={posts.id} className="PostList">
```

Comment:
The `key` attribute on the `div` element is not necessary and is incorrectly using `posts.id`, which does not exist. The `key` should be used on elements inside the `map` function to uniquely identify each child element. You can remove the `key` attribute from this `div`.